give.spec

```diff
@@ -101,6 +79,8 @@ rm -rf $RPM_BUILD_ROOT
  * Thu Nov 01 2012 Georgia Pedicini <gap@lanl.gov>
 - LANL version 3.1-2, tighten permissions
  * Tue Nov 06 2012 Georgia Pedicini <gap@lanl.gov>
-- Added defined text string to include in description, citing which (if any) options were used in the build.
+- Added defined text string to include in description, citing which (if any)
+options were used in the build.
  * Wed May 25 2016 Dominic Manno <dmanno@lanl.gov>
-- Converted to be python2 and python3 compatible, mostly print statement to function calls 
+- Converted to be python2 and python3 compatible, mostly print statement to
+function calls
```

Comment:
When we are happy with the spec file changes you will want to add a change log entry here. You want a single brief sentence that encapsulates what you have done. Also, consider using theimperative tone, e.g., `update and simplify spec file` instead of `updated and simplified [...]`